common/src/main/java/com/wynntils/services/itemfilter/ItemFilterService.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright © Wynntils 2023.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.services.itemfilter;
+
+import com.wynntils.core.components.Service;
+import com.wynntils.utils.type.ErrorOr;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import net.minecraft.client.resources.language.I18n;
+
+public class ItemFilterService extends Service {
+    private final Map<String, ItemFilterFactory> filterFactories = new HashMap<>();
+
+    public ItemFilterService() {
+        super(List.of());
+        registerFilter(new LevelItemFilterFactory());
+        registerFilter(new ProfessionItemFilterFactory());
```

Comment:
Just so we don't do this in the constructor, can you extract this to a method? Kind of like `com.wynntils.core.consumers.functions.FunctionManager#registerAllFunctions`